src/nft/components/bag/Bag.tsx

```diff
@@ -76,9 +76,9 @@ const Bag = () => {
   const setTotalUsdPrice = useBag((s) => s.setTotalUsdPrice)
 
   const { pathname } = useLocation()
-  const isProfilePage = pathname.startsWith('/profile')
+  const isProfilePage = pathname.startsWith('/nfts/profile')
   const isNFTPage = pathname.startsWith('/nfts')
-  const shouldShowBag = isNFTPage || isProfilePage
+  const shouldShowBag = isNFTPage
```

Comment:
this seems a bit redundant now we are only doing one boolean check.  Probably should replace all ```shouldShowBag``` with ```isNFTPage```

Reply:
i can do that